system-tests/runtimes/file-transfer-provider/src/main/java/org/eclipse/edc/test/extension/api/FileTransferExtension.java

```diff
@@ -38,6 +39,7 @@
 public class FileTransferExtension implements ServiceExtension {
 
     public static final String USE_POLICY = "use-eu";
+    @Setting(value = "Path to test assets")
```

Comment:
pls improve

Reply:
this is a `system-tests` runtime, I think we can avoid documenting it